### DISCUSSION TITLE
Brew returning 403 on Mac M1 Ventura 13.0.1

### DISCUSSION THREAD

**Discussion by nitingupta40** [Homebrew/discussions#4014] [Getting started]
When I run below command in Mac M1 Ventura 13.0.1 in get 403. Any help please
path  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

curl: (22) The requested URL returned error: 403

**Comment by DeveloperKev** [comment]
i am facing too

**Comment by p-linnane** [comment]
Unable to reproduce this. GitHub might have been having some issues.

**Reply by nitingupta40** [reply]
curl -fsSL  https://raw.githubusercontent.com/Homebrew/install/master/install.sh
curl: (22) The requested URL returned error: 403

Curl itself is throwing 403

**Comment by chenrui333** [comment]
```
$ curl -I https://raw.githubusercontent.com/Homebrew/install/master/install.sh
HTTP/2 200
cache-control: max-age=300
```

Maybe firewall problem?
